es-vue-base/src/lib-components/EsButton.vue

```diff
@@ -60,3 +69,15 @@ export default {
     },
 };
 </script>
+
+<style lang="scss" scoped>
+.btn-link.inline {
```

Comment:
I wonder if this should go in `es-bs-base`, but we can always extract it out later

Reply:
Possibly? I guess you could manually apply the `inline` and `btn-link` classes to a `<button>` on one of our BS2->BS4 pages - would have to confirm that's a need.